src/mudlet-lua/lua/geyser/GeyserLabel.lua

```diff
@@ -410,7 +410,7 @@ end
 --- Internal function when a parent nest element is clicked
 --- to lay out the nested elements within
 -- @param label The name of the label to use
-function doNestClick(label)
```

Comment:
Yes, its because my initial commit was done on the "development" branch, before I realized i should have made a unique branch per pull request. Now this new branch "multiselectionlistwidget" was created prior to the other pull request being merged. Needless to say I will be carefull that changes go into their own branches from now on. I am still new to using github.

Reply:
**Well** if the other PR goes in without any further changes you will get away with it because Git will (normally) silently *fast-forward* over a change that gets merged in from a different commit - so, since the change was already done it will disappear from the set of changes in *this* PR - but if the other one has to be changed than this PR will start reporting the difference as a conflict and require the code here to be made to have the same changes (you would have to *merge* the revised development` branch into your local copy of this branch/PR and then push the merged version out to your repo...) :nerd_face: